docs/usage-and-billing.mdx

```diff
@@ -0,0 +1,158 @@
+---
+slug: usage-and-billing
+append_help_link: true
+title: Usage and billing 
+description: "Learn about pricing, tiers, and feature support for the following Semgrep Products: Semgrep OSS, Semgrep Code, and Semgrep Supply Chain."
+tags:
+  - Semgrep Cloud Platform
+  - Semgrep OSS
+  - Team & Enterprise Tier
+hide_title: true
+---
+
+import MoreHelp from "/src/components/MoreHelp";
+
+<ul id="tag__badge-list">
+  {Object.entries(frontMatter)
+    .filter((frontmatter) => frontmatter[0] === "tags")[0]
+    .pop()
+    .map((value) => (
+      <li class="tag__badge-item">{value}</li>
+    ))}
+</ul>
+
+<!-- # Pricing and billing -->
+
+# Usage and billing 
+
+Learn about usage computation and other aspects of your Semgrep licenses.
+
+:::note
+- This document is for **Semgrep Pro** users; Semgrep OSS Engine does not have any usage limit. Compare [<i class="fa-regular fa-file-lines"></i>  Semgrep Pro and Semgrep OSS](/semgrep-pro-vs-oss).
+:::
+
+## Usage 
+
+All Semgrep Pro products are free under the **Team** tier for **10 monthly contributors**. These products include:
+
+- Semgrep Cloud Platform
+- Semgrep Code
+- Semgrep Supply Chain
+- Semgrep Secrets (beta)
+
+A **contributor** is someone who has made at least **one** commit to a Semgrep-scanned private repository within the last month, starting from the **date of license purchase** if a license was purchased, or the date of account creation, for accounts using Semgrep within usage limits.
```

Comment:
I think we need a little more emphasis on private here - I missed it and below got confused when I didn't think we'd mentioned it.